### ISSUE TITLE
 Unable to retrieve the list of statuses on Jira server

### ISSUE THREAD

**Issue by sipani909** [zenoss/testrail-jira-gadgets#14]
Am seeing this issue even after whitelisting the testrail URL. Does integration need to be enabled between testrail and JIRA for this gadget to work?
Debugging according to steps provided in previous ticket shows the following:
obj = {text: "", rc: 500, headers: {…}, oauthApprovalUrl: undefined, oauthError: undefined}

**Comment by AwesomeDonYoung** [comment]
No, I don't believe integration needs to be enabled. Something to try: make an API call (a simple one, like get_statuses) from a REST API client or a curl call to see if TestRail API calls are working.
